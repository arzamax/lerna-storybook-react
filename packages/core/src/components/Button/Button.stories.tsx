import { Button } from '.';

export default {
  component: Button,
  title: 'Button',
};

export const Default = () => <Button />;
