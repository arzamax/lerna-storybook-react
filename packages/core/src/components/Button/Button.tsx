import { useButtonTitle } from '@example/utils';

export const Button = () => {
  const title = useButtonTitle();

  return <button>{title}</button>;
};
