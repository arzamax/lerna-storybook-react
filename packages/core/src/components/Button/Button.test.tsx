import React from 'react';
import { render } from '@testing-library/react';

import { Button } from './Button';

const setup = () => render(<Button />);

describe('Button test', () => {
  test('snapshot', () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });
});
