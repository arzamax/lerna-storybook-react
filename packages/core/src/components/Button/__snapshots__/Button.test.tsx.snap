// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test snapshot 1`] = `
<DocumentFragment>
  <button>
    Button title
  </button>
</DocumentFragment>
`;
